packages/common/src/search/_internal/hubEventsHelpers/processFilters.ts

```diff
@@ -31,6 +45,14 @@ export function processFilters(filters: IFilter[]): Partial<GetEventsParams> {
   if (tags?.length) {
     processedFilters.tags = tags;
   }
+  const readGroups = getOptionalPredicateStringsByKey(filters, "readGroups");
```

Comment:
Our other filter names that deal with ids are more explicitly named, e.g. `eventIds`. Lets follow the same convention here for consistency.

```ts
  const readGroupIds = getOptionalPredicateStringsByKey(filters, "readGroupIds");
  if (readGroupIds?.length) {
    processedFilters.readGroups = readGroupIds;
  }
  const editGroupsIds = getOptionalPredicateStringsByKey(filters, "editGroupsIds");
  if (editGroupsIds?.length) {
    processedFilters.editGroups = editGroupsIds;
  }
```